Environments/SharedServices_OnpremisesExtension/parameters.json

```diff
@@ -99,7 +101,7 @@
         "AzureBastion": {
             "ResourceGroup": "${Parameters.ModuleConfigurationParameters.VirtualNetwork.ResourceGroup}",
             "Name": "${Parameters.InstanceName}-bastion",
-            "Location": "West US",
+            "Location": "${Parameters.Location}",
```

Comment:
The template itself has an allowedValues in the location parameter, using a location different than this will throw a validation exception in ARM, maybe we can address this later by adding a condition to only deploy Azure Bastion if the deployment location is within the allowed regions.